tests/conftest.py

```diff
@@ -63,6 +59,26 @@ def entity_factory():
     return EntityFactory()
 
 
+@fixture
+def epics_root(samples: Path, tmp_path: Path, mocker: MockerFixture):
+    # create an partially populated epics_root structure in a temporary folder
+    epics = tmp_path / "epics"
+    shutil.copytree(samples / "epics", epics)
```

Comment:
`tests/samples/epics/support/configure/RELEASE` defines `SUPPORT=/epics/support`, is that a problem?

Reply:
I think not - this drives what goes in the output st.cmd etc. but not where ibek looks for things.